soap_test.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# Try to dump SOAP vectors (features) for each atom of each molecule
+
+import sys
+import numpy as np
+from ase.io import read as ase_read
+from rascal.representations import SphericalInvariants as SOAP
+
+# What is this integer? Unit? Reasonable range to explore?
+max_radial = 6
+# What is this integer? Unit? Reasonable range to explore?
+max_angular = 6
+# what is a power spectrum?
+soap_type = "PowerSpectrum"
+#           "RadialSpectrum" # only other possibility
+if soap_type == "RadialSpectrum":
+    max_angular = 0
+
+input_fn = "reference_data/inputs/small_molecules-20.json"
+
+# only those atomic numbers in input_fn
+global_species = [1,6,7,8]
+
+# doc can be found at
+# https://github.com/lab-cosmo/librascal/bindings/rascal/representations/
+#   spherical_invariants.py
+
+hyper_params = {
+    # Angstroms? what is the reasonable range to explore for this value?
+    "interaction_cutoff": 3.5,
+    # Angstroms? what is the reasonable range to explore for this value?
+    "cutoff_smooth_width": 0.5,
+    "max_radial": max_radial,
+    "max_angular": max_angular,
+    "gaussian_sigma_type": "Constant", # only possible choice
+    # Angstroms? what is the reasonable range to explore for this value?
+    "gaussian_sigma_constant": 0.5,
+    "soap_type": soap_type,
+    # 'RadialScaling' is the only other choice
```

Comment:
RadialScaling is defined in https://pubs.rsc.org/en/content/articlelanding/2018/CP/C8CP05921G, for the QM9 example